MCP_DAC.h

```diff
@@ -74,47 +74,78 @@ class MCP_DAC
   void     reset();
   bool     usesHWSPI() { return _hwSPI; };
 
-  // ESP32 specific
-  #if defined(ESP32)
+  
+  #if defined(ESP32)                    // ESP32 specific
+
   void     selectHSPI() { _useHSPI = true;  };
   void     selectVSPI() { _useHSPI = false; };
   bool     usesHSPI()   { return _useHSPI;  };
   bool     usesVSPI()   { return !_useHSPI; };
 
-  // to overrule ESP32 default hardware pins
+  // to overrule the ESP32s default hardware pins
+  void     setGPIOpins(uint8_t clk, uint8_t miso, uint8_t mosi, uint8_t select);
+
+  
+  #elif defined(ARDUINO_ARCH_RP2040)    // RP2040 specific
+
+  // check which SPI-Bus (SPI or SPI1) is used
+  void     selectSPI()  { _useSPI = true;   _useSPI1 = false;  };
```

Comment:
One variable should be enough to indicate which SPI is used?
(compare the ESP32   _useHSPI  flag